tests/integration_tests/test_gas.py

```diff
@@ -24,14 +17,34 @@ def test_equivalent_gas_consumption(geth, ethermint):
     # ensure that the gasUsed is equivalent
     assert geth_reciept.gasUsed == ethermint_reciept.gasUsed
 
-    w3_wait_for_new_blocks(geth.w3, 5)
-    w3_wait_for_new_blocks(ethermint.w3, 5)
 
-    # repeat the above process with contract deployment
+def test_gas_deployment(geth, ethermint):
+    # deploy an identical contract on geth and ethermint
+    # ensure that the gasUsed is equivalent
     _, geth_contract_reciept = deploy_contract(
         geth.w3,
         CONTRACTS["TestERC20A"])
     _, ethermint_contract_reciept = deploy_contract(
         ethermint.w3,
         CONTRACTS["TestERC20A"])
     assert geth_contract_reciept.gasUsed == ethermint_contract_reciept.gasUsed
+
+
+def test_gas_call(geth, ethermint):
+    function_input = 10
+
+    # deploy an identical contract on geth and ethermint
+    # ensure that the contract has a function which consumes non-trivial gas
+    geth_contract, _ = deploy_contract(
+        geth.w3,
+        CONTRACTS["BurnGas"])
+    ethermint_contract, _ = deploy_contract(
+        ethermint.w3,
+        CONTRACTS["BurnGas"])
+
+    # call the contract locally (eth_call) and compare gas estimates
+    geth_estimated_gas = (geth_contract.functions
```

Comment:
Dont just call the `estimateGas` function but also do the execution. This is important as it will actually go throught the gasHandler